engine/lib/configuration.php

```diff
@@ -88,12 +87,11 @@ function elgg_get_engine_path() {
  * Get an Elgg configuration value
  *
  * @param string $name      Name of the configuration value
- * @param int    $site_guid null for installation setting, 0 for default site
- *
+ * @param int    $site_guid null for installation setting anything else for current site
  * @return mixed Configuration value or null if it does not exist
  * @since 1.8.0
  */
-function elgg_get_config($name, $site_guid = 0) {
+function elgg_get_config($name, $site_guid = null) {
```

Comment:
did that before...  think it is better to leave as is... internally the get function acts different using NULL vs any int. NULL gets from datalist... ints from ConfigTable.. did not see a way to do this in an easy way... any idea?